m2cgen/VERSION.txt

```diff
@@ -1 +1 @@
-0.9.1
+0.10.1
```

Comment:
Ah, I'm very sorry! I forgot to bump the version for development last time.
Thanks for fixing this!

Reply:
No problem at all!

Reply:
I think we need a better process for this anyway.